cmd/login.go

```diff
@@ -138,6 +145,11 @@ func randToken() string {
 	return base64.StdEncoding.EncodeToString(b)
 }
 
+func showAuthorizationURL() {
```

Comment:
this function is missing the parameter.
```suggestion
func showAuthorizationURL(authorizationURL string) {
```